instances/latest/terminologies/disease/coronavirusDisease2019.jsonld

```diff
@@ -2,15 +2,17 @@
   "@context": {
     "@vocab": "https://openminds.ebrains.eu/vocab/"
   },
-  "@id": "https://openminds.ebrains.eu/instances/disease/coronavirusDisease",
+  "@id": "https://openminds.ebrains.eu/instances/disease/coronavirusDisease2019",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Disease",
-  "definition": "A Coronavirus infectious disease that is characterized by fever, cough and shortness of breath and that has_material_basis_in SARS-CoV-2.",
-  "description": "Coronavirus disease 2019 (COVID-19) is an infectious disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)",
+  "definition": "An infectious disease caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), and typically presents with fever, cough and shortness of breath.",
```

Comment:
A coronavirus infectious disease that is characterized by fever, cough and shortness of breath and caused by the coronavirus strain SARS-CoV-2.